src/cljs/rems/modal.cljs

```diff
@@ -0,0 +1,71 @@
+(ns rems.modal
+  (:require [rems.guide-functions]
+            [rems.collapsible :as collapsible]
+            [rems.text :refer [text]])
+  (:require-macros [rems.guide-macros :refer [component-info example]]))
+
+(defn- shade-wrapper [content on-close]
+  [:div.modal--shade
+   {:style {:position :fixed
+            :z-index 10000
+            :left 0
+            :top 0
+            :right 0
+            :bottom 0
+            :background-color "rgba(0,0,0,0.5)"
+            :display :flex
+            :justify-content :center
+            :align-items :center}
+    :on-click on-close}
```

Comment:
This click handler grabs too many events. It triggers also when you click inside the dialog, so e.g. it's not possible to select and copy the error message.

Reply:
Added stopPropagation